test/metabase/test/generate.clj

```diff
@@ -275,6 +275,10 @@
     (= :field ent-type)
     (update :name unique-name)
 
+    ;; [Field ID, Dimension name] pairs need to be unique. This enforces it, and appends junk to names if needed.
+    (= :dimension ent-type)
```

Comment:
This is a [Yoda condition](https://en.wikipedia.org/wiki/Yoda_conditions).

Consider two ways to check what sort of bird we have on hand:

1. `(= bird-type :toucan)` -- one reads this code as "bird type equals toucan"
2. `(= :toucan  bird-type)` -- you'd read this as "toucan equals bird type"

Now while they both do exactly the same thing, the former is the "normal" way to write things and the latter is a Yoda condition.